.github/workflows/fmt.yml

```diff
@@ -0,0 +1,23 @@
+name: fmt
```

Comment:
You can just do this on the build one, before the actual build. So we make sure if it doesn't follow formatting, it actually fails to build.

Reply:
formatting and linting don't really affect features much so they are generally kept separate from the main build pipeline

Reply:
Agreed it doesn't affect, my reasoning goes to having code standards. The code either follows the pattern or it doesn't, if it doesn't then we shouldn't build just for the sake to push better standards.

Reply:
in that case you can use branch protection rules and block pr merging if `fmt` doesn't pass.

Reply:
Yep, that can be done.